### PR TITLE
Add database support to plantUML

### PR DESCRIPTION
**Pull request by tfij**
Issue: #1132

Please review this change. If it is ok I'll add more tests.

### CODE REVIEW

File: archunit/src/main/java/com/tngtech/archunit/library/plantuml/rules/PlantUmlPatterns.java [hunk 1]
```diff
@@ -36,17 +37,20 @@
...
     private static final String STEREOTYPE_FORMAT = "(?:<<" + capture(anythingBut("<>")) + ">>\\s*)";
     private static final Pattern STEREOTYPE_PATTERN = Pattern.compile(STEREOTYPE_FORMAT);
-
     private static final String ALIAS_GROUP_NAME = "alias";
     private static final String ALIAS_FORMAT = "\\s*(?:as \"?" + capture("[^\" ]+", ALIAS_GROUP_NAME) + "\"?)?";
 
     private static final String COLOR_FORMAT = "\\s*(?:#" + anyOf("\\w|/\\\\-") + "+)?";
-
+    private static final String COMPONENT_TYPE_GROUP_NAME = "componentType";
+    private static final String COMPONENT_TYPE_FORMAT = capture("component", COMPONENT_TYPE_GROUP_NAME) + "?";
```
Comment (tfij) [inline archunit/src/main/java/com/tngtech/archunit/library/plantuml/rules/PlantUmlPatterns.java:48]: this is optional and lib should support both
```
[Comp1] <<..c1..>>
```
and
```
component [Comp1] <<..c1..>>
```
Currently it support only first option.

**Comment by tfij** [comment]
hi @codecholeric 
waiting for your feedback about the change. Then I'll apply your suggestions and add more tests.

**Comment by tfij** [comment]
@codecholeric and @hankem
how can I help you in reviewing this code?

**Review by bgalek — APPROVED** [review]

**Comment by codecholeric** [comment]
Superseded by #1184

**Pull request closed without merging**